src/services/index.ts

```diff
@@ -1,4 +1,5 @@
-import * as Keychain from 'react-native-keychain'
```

Comment:
If we use another module instead of Keychain, maybe need to delete keychain from package.json?

Reply:
Good catch. But I have used `react-native-encrypted-storage` package. It uses both `react-native-keychain` (native storage for iOS) and `react-native-shared-preferences` (native storage for Android). And it automatically download them.